README.md

```diff
@@ -1 +1,18 @@
-﻿# Balanced-Management
+﻿# Automated-Compliance
+
+I'm a college student working on a complex project related to regulatory compliance at an insurance company.
+The problem concerns the difficulty of ensuring uniform and consistent application of regulations in all financial transactions and contracts. 
```

Comment:
No so cosa vuoi fare con questo, ma non corresponde di nulla a quello che cerchiamo nella classe. Deve essere
1. Problema personale, qualsiasi roba che ti piacerebbe risolvere
2. Devi avere dati per risolverlo, non fare dati finti
3. Scrivere tutto il codice che serve per risolvere il problema.

Non so cosa sono i regulazione di sicuranza. Non so cosa sono contratti commerciali. E chi sono tutti quelli. Ma mi sembra quasi sicuro que non c'entra niente con quello che andiamo cercando qui.
E quello che cerchiamo chi è, di solito, l'ottimo per lo studente per imparare concetti complessi della engegneria del software.
Qui ho proposto di fare un giocco per applicare "design thinking" al dissegno di un problema. Sempre in una engegneria del software si empiagano metodologie che non assicurano l'ottimalità, ma che fanno sicuro che si trova una soluzione al problema in tempo e forma. Se non se fa così, potete fare una serie infinita de "problemi", che non trovarei uno adatto a la lezione.
È per quello che ti dico: cerca di capire cosa è *design thinking* e perchè e bravo per ciò. Puoi farlo con il giocco (se volete, lo facciamo un'altra volta il giovedì) o da solo, è lo stesso. Ma dovete capire che l'unica maniera di risolvere problema in engegneria è empiegare metodologie e buone pratiche que sono adatte a particulari fasi d'un problema, come qui.

Reply:
@JJ  Ho scritto un altro problema, se non dovesse andare bene neanche questo problema, io oggi continuerò a scrivere altri problemi e domani mattina li presenterò a lezione